### PR TITLE
RP intro add tutorial link

### LINKED ISSUE #4762: Create a web app is hard to follow
Some customers find https://docs.microsoft.com/en-us/aspnet/core/mvc/razor-pages/ hard to follow. Add a note saying it's not a tutorial, detailed instructions can be found in https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start

Add that as next step as the end of the tutorial.

Verbatim:

>Am I the only one who is confused by the fact that there are no instructions on what files I'm supposed to place the data model and db context in?

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #4762
[Review URL1](https://review.docs.microsoft.com/en-us/aspnet/core/mvc/razor-pages/?branch=pr-en-us-4764)
[Review URL1](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start?branch=pr-en-us-4764)

### CODE REVIEW

File: aspnetcore/mvc/razor-pages/index.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ Razor Pages is a new feature of ASP.NET Core MVC that makes coding page-focused
 
 If you're looking for a tutorial that uses the Model-View-Controller approach, see [Getting started with ASP.NET Core MVC](xref:tutorials/first-mvc-app/start-mvc).
 
+This document provides an introduction to Razor Pages, it is not a step by step tutorial. If you find some of the sections difficult to follow, see [Getting started with Razor Pages](xref:tutorials/razor-pages/razor-pages-start).
```
Comment (scottaddie) [inline aspnetcore/mvc/razor-pages/index.md]: ~~This document provides an introduction to Razor Pages, it is not a step by step tutorial.~~
This document provides an introduction to Razor Pages. It's not a step-by-step tutorial.

File: aspnetcore/mvc/razor-pages/index.md [hunk 2]
```diff
@@ -409,5 +411,6 @@ services.AddMvc()
 
 ## See also
 
+* Getting started with Razor Pages](xref:tutorials/razor-pages/razor-pages-start).
```
Comment (guardrex) [inline aspnetcore/mvc/razor-pages/index.md]: Stray period at the end?
